cves/CVE-2016-6817.yml

```diff
@@ -204,4 +204,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: 
+  In my opinion, this was a coding mistake. This specific vulnerability is based off an edge-case that, at the time, 
+  developers did not think would ever occur. The founder of the vulnerability, barney2k7, discovered 
+  such edge case when trying to parse a header about 30kb. The fix of this vulnerability involves 
+  increasing the buffer size if the capacity were to be less than the size of the header. The CWE entry for this 
+  has mitigations which the fix follows. The fix checks to make sure if the size is larger than the capacity 
+  and if it is, increase the capacity. 
```

Comment:
`parse a header about 30kb.`
Insert 'of' to make `parse a header of about 30kb`